apps/web/lib/validators/escrow.ts

```
@@ -0,0 +1,95 @@
+    if (!milestone.title?.trim()) {
+        errors.push(`Milestone ${index + 1}: Title is required`);
+    }
+    if (!milestone.description?.trim()) {
+        errors.push(`Milestone ${index + 1}: Description is required`);
+    }
+    if (typeof milestone.amount !== "number" || milestone.amount <= 0) {
+        errors.push(`Milestone ${index + 1}: Amount must be a positive number`);
+    }
+    if (
+        !(milestone.dueDate instanceof Date) &&
+        isNaN(Date.parse(String(milestone.dueDate)))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)